packages/sit-onyx/src/components/OnyxDialog/OnyxDialog.stories.ts

```diff
@@ -33,3 +35,27 @@ export const Default = {
     default: "Dialog content...",
   },
 } satisfies Story;
+
+export const Modal: Story = {
```

Comment:
you could add a description where you recommend to close it with Escape after opening, if someone thinks they are getting stuck on the docs page ^^